### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by joe513**
also #27 fix.

### CODE REVIEW

File: app/admin.py [hunk 1]
```diff
@@ -5,18 +5,21 @@
 from app.models import Channel, Playlist, Upload
 
 
+@admin.register(Channel)
```
Comment (inoks) [inline app/admin.py]: What is the benefits of it?

File: app/tests.py [hunk 2]
```diff
@@ -10,16 +10,16 @@
...
-        self.email = 'john@example.com'
-        self.password = 'dolphins'
+        self.username, self.password, self.email = 'John Doe', 'dolphins', 'john@example.com'
+
         self.user = User.objects.create_user(
             username=self.username,
             email=self.email,
             password=self.password
         )
-        self.client = Client()
```
Comment (inoks) [inline app/tests.py]: Why you removed it?

Comment (joe513) [reply app/tests.py]: @inoks please take a little look at [this](https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.SimpleTestCase.client) very short explanation.
TestCase inherit SimpleTestCase. So TestCase has self.client. It was especially realesed for testing purposes (not to declare this) so we can just use the build-in client, it's the same as you have declared.

You can also take a look at this simple photo to be sure TestCase inherits SimpleTestCase.

File: app/admin.py (hunk 1 above)
Comment (joe513) [reply app/admin.py]: idk, but it looks like nicer. I can return an old style if you want.

File: app/tests.py (hunk 2 above)
Comment (inoks) [reply app/tests.py]: Ok, that make sense now.

**Pull request closed without merging**